docs/pages/blog/2021-q2-update.md

```diff
@@ -0,0 +1,221 @@
+
+#### Data Grid
+
+- 🚨 We have continued to work on breaking changes to reach a stable version.
+  We think that the component is good enough for usage in production, but we want to get the public API right before committing to a stable release.
+- ✍️ We have added support for the [cell edit mode](https://material-ui.com/components/data-grid/editing/).
+
+  <img loading="lazy" src="/static/blog/2021-q2-update/cell-edit.gif" alt="" style="width: 842px; margin-bottom: 16px;" />
+
+- 🐛 We have focused on fixing bugs and regressions to ensure early users have a great experience with the component.
+  This was done instead of taking on new ambitious features.
+  We want our users to be able to adopt new features as fast as we build them.
+- 🔘 We have added support for a built-in [single select](/components/data-grid/columns/#column-types) column type:
+
+  <img loading="lazy" src="/static/blog/2021-q2-update/single-select.png" alt="" style="width: 481px; margin-bottom: 16px;" />
+
+  <p class="blog-description">the <a href="https://codesandbox.io/s/material-demo-forked-iuyo5?file=/demo.js">codesandbox</a></p>
+
+  and the **boolean** column type.
+
+- 🚀 We have improved the performance.
+  Client-side sorting and filtering are x2-3 faster on large data sets.
+  We have improved the UX when scrolling.
+  We have added a memo logic on the render cell's parent and work on its effectiveness to reduce the rendering of custom cells.
+  We have identified more opportunities to improve the performance, but we won't prioritize them just yet.
+- 📚 We have worked on providing more reference documentation. We are semi-automatically generating it from the TypeScript source definitions, with the descriptions.
+  For instance, you can now find all the [properties available](/api/data-grid/grid-col-def/) of the `GridColDef`.
+- ⚡️ We have fixed support for components that use portals, like Select and Autocomplete in the cell editing mode.
+- 🌏 We have accepted many new built-in locales (+15) from the community, after the introduction of [the feature](/components/data-grid/localization/#supported-locales) in Q1.
+
+### Design kits
+
+#### Figma
+
+Designers have a great sense of detail.
+We have polished the kit, using all the feedback that we could get.
+We have also introduced support of a [dark mode](https://www.figma.com/file/7M8OmZIv6WigOjbSV7Xxlg/2021-q2-update?node-id=4230%3A724)! We have worked on making the design kit close to the React components.
+This reduces miscommunications between designers and developers.
+
+#### Adobe XD
+
+We have fixed performance issues and polished the kit.
+
+#### Sketch
+
+We have added support of a dark mode and polished the kit.
+
+## Company
+
+In the first quarter, we have focused on introducing a [handbook](https://material-ui.com/blog/2021-q1-update/#handbook) to share our culture and to solve the N(N-1)/2 communication channels problem that comes with a larger team.
```

Comment:
```suggestion
In the first quarter, we focused on introducing a [handbook](https://material-ui.com/blog/2021-q1-update/#handbook) to share our culture and to solve the N(N-1)/2 communication channels problem that comes with a larger team.
```

Reply:
Why do we mention this here if it was introduced in the first quarter. Maybe leave the "In the first quarter" part, if it was not mentioned before.

Reply:
@mnajdova I have used this as a transition, to explain where we are coming from.